[DTM] Updated Localization info (#1183)
SoftwareAG/c8y-docs: content/dtm/localization.md

Once you create an asset property or an asset model, their names are added automatically as keys in the **Localization** page. You can further add the translations as required.

{{< c8y-admon-info>}}

> Note that if an asset property or asset model is present, the corresponding entry on the Localization page cannot be deleted. However, if no translation exists for the asset property or asset model, deleting the asset property or asset model will automatically remove the corresponding entry.

Reviewer: ```suggestion
Note that if an asset property or asset model is present, the corresponding entry on the **Localization** page cannot be deleted. However, if no translation exists for the asset property or asset model, deleting the asset property or asset model will automatically remove the corresponding entry.
```

> On the other hand, if a translation exists for the asset property or asset model, deleting it will not automatically remove the corresponding entry. In such cases, a delete option will be available for that specific translation, allowing manual deletion if desired.

Reviewer: ```suggestion
On the other hand, if a translation exists for the asset property or asset model, deleting it will not automatically remove the corresponding entry. In such cases, a delete option will be available for that specific translation, allowing you to manually delete the entry.
```
